### PR TITLE
Use apidoc directive in general/stream/stream-design.md (#22904)

### PR DESCRIPTION
**Pull request by Captain1653**
References #22904

### CODE REVIEW

File: akka-docs/src/main/paradox/general/stream/stream-design.md [hunk 1]
```diff
@@ -90,11 +90,11 @@ Exceptions from this need to be well-justified and carefully documented.
 
 Akka Streams must enable a library to express any stream processing utility in terms of immutable blueprints. The most common building blocks are
 
- * Source: something with exactly one output stream
```
Comment (Captain1653) [inline akka-docs/src/main/paradox/general/stream/stream-design.md:93]: `handleWith` mentioned above. There are many overloaded methods `handleWith`. Which should I take? Is it important to add link?

Comment (johanandren) [reply akka-docs/src/main/paradox/general/stream/stream-design.md:93]: No, let's skip that.

**Review by johanandren — APPROVED** [review]
LGTM

**Comment by Captain1653** [comment]
Can it be reviewed for 2.6.19? Please :)

**Review by raboof — APPROVED** [review]

**Pull request merged**